### ISSUE TITLE
Add diaper changing table as POI category

### ISSUE THREAD

**Issue by ltog** [osmandapp/Osmand#5695]
I'd like to have the possibility to search for [diaper changing tables](https://wiki.openstreetmap.org/wiki/Key:diaper) as a POI category. That should work identically to searching for e.g. libraries: Tapping the magnifying glass symbol, entering `library` (or `diaper` in my case) and seeing the results highlighted as orange markers.

As far as I could understand, https://github.com/osmandapp/Osmand/issues/2820 is not a duplicate, but talks about the key `diaper=*` being visible when tapping a given object and then tapping the "Description" button.

**Comment by scaidermern** [comment]
I guess the actual POI type doesn't really matter when being in need for this. Displaying all POIs with `diaper=*` (except for `diaper=no`) should be a reasonable approach.

**Comment by ltog** [comment]
> I guess the actual POI type doesn't really matter when being in need for this. Displaying all POIs with diaper=* (except for diaper=no) should be a reasonable approach.

That's exactly my point of view too.

**Comment by scaidermern** [comment]
Unfortunately there are two tags to look out for: `diaper=*` and `toilets:changing_table=yes`.

**Comment by ltog** [comment]
@scaidermern: Good catch! However, according to Taginfo `toilets:changing_table=` is nearly unused.

**Comment by vshcherb** [comment]
I took a look as well,
poi_additional  means it is not a separate category but it is indexed and searchable. Though you need to configure filter in category service or in toilets for example. So it is an additional information and not POI itself.

**Comment by bkil** [comment]
There is an ongoing proposal to enhance tagging of baby changing tables. Please share with whoever may be concerned and feel free to comment:
https://wiki.openstreetmap.org/wiki/Proposed_features/baby_changing_tables

**Comment by midnightcomm** [comment]
I would really appreciate being able to find a [changing table](https://wiki.openstreetmap.org/wiki/Key:changing_table) for diapers/nappy. Either by searching POIs for "changing table", or finding "Toilets" and having a filter for "changing table".

Often places that have a changing table will only have them in the women's room, which as a father can be difficult. Being able to select the location of a changing table by male,female,unisex rooms would be useful to me, but may prove excessively complicated to the average user.
